templates/config/config_pem.html

```diff
@@ -5,15 +5,16 @@
   <!-- <label for="Name" class="col-sm-2 col-form-label">Name</label> -->
   <form class="form-inline" action="{{ url_for('pem_create') }}" method="POST">
     <div class="col-sm-12">
-        <input type="text" id="Name" name="Name" class="form-control" placeholder="Insert Key Pair Name"/>
-        <button type="submit" class="btn btn-primary">CREATE</button> 
+        <input type="text" id="pemName" name="Name" class="form-control" placeholder="Insert Key Pair Name"/>
+        <button type="submit" class="btn btn-primary">CREATE</button>
     </div>
   </form>
 </div>
 <div class="table-responsive">
-  <table class="table table-striped table-bordered table-hover table-sm" id="pem-table">
+  <table class="table table-striped table-bordered table-hover table-sm" id="pemTable">
```

Comment:
HTML id and class doesn't have to be camalCase. check the class name right beside id.